brave/brave-core
# feat(wallet): Add more transak tokens, fix WETH

Resolves https://github.com/brave/brave-browser/issues/29055



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Go to Fund wallet screen
- Search for the following tokens:
    - LOOKS(Ethereum)
    - FRAX(Ethereum)
    - FOOD(Polygon)
    - WETH(Polygon)
- Click select buy option
- Transak should be show in buy option list
